blobfuse/blobfuse.cpp

```diff
@@ -191,15 +282,92 @@ int read_config(const std::string configFile)
 
 void *azs_init(struct fuse_conn_info * conn)
 {
+    const int defaultMaxConcurrency = 20;
```

Comment:
This is probably on me, but we should move this to constants and replace all the magic 20s with this value.